From bbeb9d73191c1c79f5286fd3b1fdb5611fb11301 Mon Sep 17 00:00:00 2001
From: "Patrick J. Volkerding" <volkerdi@slackware.com>
Date: Sun Apr 26 18:51:23 CDT 2020
Subject: [PATCH] Switch from tempfile to mktemp

Use the standard mktemp utility from GNU coreutils rather than the
lesser-known tempfile utility from Debian of yore.

--- mkinitrd.orig	2019-09-04 23:37:08.000000000 -0500
+++ mkinitrd	2020-04-26 18:46:22.352636100 -0500
@@ -235,7 +235,7 @@
 
   # Then copy all remaining libs our initrd files link against:
   COUNT=1
-  PRFX=$(tempfile --prefix ldd-)
+  PRFX=$(mktemp --suffix -ldd)
   TMPFILE=${PRFX}${COUNT}
 
   find $SOURCE_TREE -type f -exec ldd {} 2>/dev/null \; | unify_libs > $TMPFILE
